script/Arcueid/Day 4/04_00_ARC04_2.txt

```diff
@@ -21,30 +21,33 @@ Kohaku-san comes in, vigorously pushing a servant cart.
 [sha:aac31667b2533c0caa053e7db01a0d47194e7c4b]{
 -- Page 96, Offset 5804.
 -- 時間もないし、チャチャっと食っちまおう」
-We don't have much time, so let's just eat quickly."
+I don't have much time, so I'd better scarf this fast."
 }
 [sha:bc82379a7fc428805c732d6837cfd0caa76dc8f8]{
 -- Page 97, Offset 5805.
 -- 「兄さん。その乱暴な言葉<遣|づか>いはやめてください」
-"Nii-san, please stop using such rough language."
+"Nii-san, please stop using such coarse language."
 }
 [sha:f9f15492d2faeecae0b37dd1af09582ebd6ae36a]{
 -- Page 97, Offset 5806.
 -- 「ありゃ、すっかりもとの調子に戻ったか。俺の心配をしてくれていた方がおとなしくて良かったのに」
-"Hey, you're all back to normal. It might've been better if you had stayed worried about me."
+"Hey, you're all back to normal.
+ I liked you better when you were shyly fretting over me."
```

Comment:
do not like "shy" here. I think it's more like "obediently" or "docilely" if you don't like the ordering connotation